SD-K9/src/app/components/building-info/building-info.component.ts

```diff
@@ -69,7 +71,10 @@ export class BuildingInfoComponent implements OnInit, OnDestroy {
 
   private _displayCard() {
     if (this.currentCampus !== '' && this.currentBuilding !== '') {
-      this.currentBuildingInfo = this._campusConfig[this.currentCampus]["buildings"][this.currentBuilding]['info'];
+      this.currentBuildingDepartmentsInfo = this._campusConfig[this.currentCampus]["buildings"][this.currentBuilding]['departments'];
+      this.currentBuildingServicesInfo = this._campusConfig[this.currentCampus]["buildings"][this.currentBuilding]['services'];
+      if(this.currentBuildingDepartmentsInfo == undefined) this.currentBuildingDepartmentsInfo = "no information to show";
+      if(this.currentBuildingServicesInfo == undefined) this.currentBuildingServicesInfo = "no information to show";
```

Comment:
I know it looks chunky, but since we have been following this syntax of if statement, I think its best to stay consistent.
```suggestion
      if (this.currentBuildingDepartmentsInfo == undefined) {
        this.currentBuildingDepartmentsInfo = "no information to show";
      }
      if (this.currentBuildingServicesInfo == undefined) {
        this.currentBuildingServicesInfo = "no information to show";
      }
```